### PR TITLE
doc: added instructions for semantic commits in readme.md file

### PR DESCRIPTION
**Pull request by rohitdasu**
### 👨‍💻 Changes proposed

Added instructions for semantic commits

### ✔️ Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

### 📄 Note to reviewers

Added instructions for semantic commits as discussed with @ykdojo

### CODE REVIEW

**Comment by rohitdasu** [comment]
> Someone is attempting to deploy a commit to a Personal Account owned by @ykdojo on [Vercel](https://vercel.com).
> 
> @ykdojo first needs to [authorize it](https://vercel.com/git/authorize?type=github&job=%7B%22headInfo%22%3A%7B%22sha%22%3A%227869061d441b0fd0100826a2010c2e536b2cdab2%22%7D%2C%22id%22%3A%22QmUGNUgwwAhJJSqUfKhSQB1y8iXCPfytBUanDPuRYoN8v5%22%2C%22org%22%3A%22ykdojo%22%2C%22prId%22%3A47%2C%22repo%22%3A%22defaang%22%7D).

@ykdojo could you please what's wrong with the crazy bot? 😆

**Comment by ykdojo** [comment]
@rohitdasu haha I don't know! I should probably contact Vercel to see what's going on

**Comment by ykdojo** [comment]
more discussion on this here: https://github.com/ykdojo/defaang/issues/37#issuecomment-1214198843

for the vercel thing, I created a new issue here: #50

**Comment by rohitdasu** [comment]
@ykdojo i've updated the PR please have a look at it.

File: README.md [hunk 1]
```diff
@@ -23,7 +23,24 @@ Even if you're an advanced coder, feel free to take one of them.
...
+
+```bash feat: (new feature for the user, not a new feature for build script)
+feat: (adding new feature)
+fix: (bug fix for the user, not a fix to a build script)
+docs: (changes to the documentation)
+style: (formatting, missing semi colons, etc; no production code change)
+refactor: (refactoring production code, eg. renaming a variable)
+test: (adding missing tests, refactoring tests; no production code change)
+chore: (updating grunt tasks etc; no production code change)
+dx: (dev experience; whatever helps to improve developer's experience and performance, comes under dx)
```
Comment (ykdojo) [inline README.md:37]: one small suggestion - let's put dx above chore

**Review by ykdojo — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (rohitdasu) [reply README.md:37]: sure

**Comment by rohitdasu** [comment]
@ykdojo done!

**Review by ykdojo — APPROVED** [review]

**Comment by ykdojo** [comment]
Looks good, thank you!

**Pull request merged**